README.md

```diff
@@ -1,10 +1,204 @@
 # Planetcantile
 
+[![PyPI version](https://badge.fury.io/py/planetcantile.svg)](https://badge.fury.io/py/planetcantile)
+[![Python Versions](https://img.shields.io/pypi/pyversions/planetcantile.svg)](https://pypi.org/project/planetcantile/)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-Adds TileMatrixSets for other planets.
-Aims to be home to a wide range of tilesets for the Moon, Mars, and more.
+Planetcantile provides TileMatrixSets (TMS) for planetary bodies throughout the solar system, enabling tiled map visualization and analysis for the Moon, Mars, and many other celestial bodies using standard web mapping techniques.
 
-Integrates with [morecantile](https://github.com/developmentseed/morecantile), ideally should allow downstream projects to benefit from additional TMSs defined here.
+A TileMatrixSet defines how a spatial area (like a planet) is divided into a hierarchy of tiles at different zoom levels. By implementing the [OGC TileMatrixSet standard](https://docs.ogc.org/is/17-083r4/17-083r4.html) for planetary bodies, Planetcantile makes it possible to use Earth-focused web mapping tools with data from across the solar system.
 
-Definitions go into the generate.py script to have fine grain control, alternatively json files can just be added directly to the data directory.
+## Table of Contents
+- [Features](#features)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+  - [Web API](#option-1-using-the-web-api)
+  - [Python Library](#option-2-as-a-python-library)
+  - [Environment Variables](#environment-variables)
+  - [Extending with Custom Definitions](#extending-with-custom-definitions)
+- [Supported Planetary Bodies](#supported-planetary-bodies)
+- [Integration with Other Libraries](#integration-with-other-libraries)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Features
+
+- <strong>Extensive Coverage:</strong> Support for ~100 celestial bodies including planets, moons, asteroids, and comets
+
+- <strong>Multiple Projection Types:</strong>
+  - Geographic (regular lat/lon) - Standard equirectangular/plate carrée projection
+  - Equidistant Cylindrical - Cylindrical projection preserving distances along meridians
+  - Web Mercator - Modified Mercator projection commonly used in web mapping
+  - Mercator - Classic Mercator projection
```

Comment:
hmm would like this be a bit more specific, technically these are "World Mercator (Mercator (variant A))" EPSG 9804 https://epsg.io/9804-method codes.